### ISSUE TITLE
How to open a split in the right side?

### ISSUE THREAD

**Issue by sansyrox** [weirongxu/coc-explorer#361]
![Screenshot 2021-01-07 at 4 08 16 PM](https://user-images.githubusercontent.com/29942790/103883156-d5949100-5102-11eb-9eb3-4bd22fcfdd59.png)

Right now, when I open a file using the explorer using the split command `s`. The left window is split into horizontal parts.

Is it possible to split the right window instead of the left one as this is how my tiling manger works and the opposite split is a little off putting.

I would really appreciate if you can point me to that config.

**Comment by weirongxu** [comment]
No, at least there is no such feature right now, if you really want to use this feature immediately, you can only use Vim API to get the node path and split vim window by yourself.

https://github.com/weirongxu/coc-explorer/wiki/Vim-API#explorergetnodeinfo

**Comment by sansyrox** [comment]
Thank you @weirongxu ! Is there a CocAction which is used to Split the window?

**Comment by weirongxu** [comment]
The built in CocAction shouldn't be used for you, you need to use vim's `split` command to execute

**Comment by ehyyoj** [comment]
Same problem as me. I am use `s` to split into horizontal before, but now is not support 😢
